osdep/win32/smtc.cc

```diff
@@ -0,0 +1,391 @@
+/*
+ * User language lookup for generic POSIX platforms
+ *
+ * This file is part of mpv.
+ *
+ * mpv is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * mpv is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "smtc.h"
+
+#include <chrono>
+#include <format>
+
+#include <Windows.h>
```

Comment:
`windows.h`?
https://github.com/mingw-w64/mingw-w64/blob/master/mingw-w64-headers/include/windows.h

I get error when building mpv with this pr:
```
/osdep/win32/smtc.cc:25:10: fatal error: 'Windows.h' file not found
```

Reply:
changed.